### PR TITLE
Add Metadata Store v1.1.0 Grype result release notes

### PR DESCRIPTION
**Pull request by xtreme-jason-smith**

### CODE REVIEW

**Review by xtreme-debbie-chen — APPROVED** [review]

File: scst-store/sca_scans/sca-scanning-results.md [hunk 1]
```diff
@@ -1,4 +1,35 @@
 # SCA scanning results
+## <a id='110'></a>1.1.0
+
+### <a id='grype-sr'></a>Grype
```
Comment (xtreme-debbie-chen) [inline scst-store/sca_scans/sca-scanning-results.md:16]: Are we no longer showing scan results for Black Duck?

File: scst-store/sca_scans/sca-scanning-results.md [hunk 2]
```diff
@@ -1,4 +1,35 @@
...
+#### <a id='kube-rbac-proxy-ci'></a>Kube RBAC Proxy Container Image
+
+* Date: March 7, 2022
+* Results: 3 high vulnerabilities: [CVE-2022-21698](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-21698), [CVE-2015-5237](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2015-5237), [CVE-2021-22570](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-22570)
+
+#### <a id='postgres-database-ci'></a>Postgres Database Container Image
+
+* Date: March 7, 2022
+* Results: 1 high vulnerability in 3 different packages: [CVE-2022-24407](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-24407), and multiple medium and low vulnerabilities. For more information see the [Postgres Database Container Image v1.1.0 CycloneDX file content](postgres-database-image-v1.1.0-grype-result.md)
+
```
Comment (julia-pu) [inline scst-store/sca_scans/sca-scanning-results.md:50]: The init container is missing here, I saw that in the jira story you've scanned it though, do we not want to mention the init container being scanned here?

File: scst-store/sca_scans/sca-scanning-results.md (hunk 1 above)
Comment (julia-pu) [reply scst-store/sca_scans/sca-scanning-results.md:16]: Black duck scan is in another story [META-513](https://jira.eng.vmware.com/browse/META-513)

Comment (xtreme-jason-smith) [reply scst-store/sca_scans/sca-scanning-results.md:16]: The black duck scans will be added in a follow up PR.

Comment (Pivotal-Jeff-Jun) [reply scst-store/sca_scans/sca-scanning-results.md:16]: It's a separate story, I have a commit for it.

Comment (xtreme-jason-smith) [reply scst-store/sca_scans/sca-scanning-results.md:16]: Well, never mind that, looks like Jeff will be updating this PR to include Black Duck scans.

File: scst-store/sca_scans/sca-scanning-results.md (hunk 2 above)
Comment (xtreme-jason-smith) [reply scst-store/sca_scans/sca-scanning-results.md:50]: Good catch, thanks. I'll add it.

**Review by julia-pu — APPROVED** [review]

**Review by Manifaust — APPROVED** [review]

**Review by Manifaust — APPROVED** [review]

**Pull request merged**